modin/experimental/xgboost/test/test_xgboost.py

```diff
@@ -0,0 +1,300 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+
+import pytest
+
+import modin
+import modin.experimental.xgboost as xgb
+import modin.pandas as pd
+import numpy as np
+import xgboost
+
+from sklearn.datasets import (
+    load_iris,
+    load_diabetes,
+    load_digits,
+    load_wine,
+    load_breast_cancer,
+)
+from sklearn.metrics import accuracy_score
+
+import multiprocessing as mp
+
+modin.config.NPartitions.put(200)
+
+
+num_cpus = mp.cpu_count()
+
+
+@pytest.mark.parametrize(
+    "type_modin_y",
+    [pd.DataFrame, pd.Series],
+)
+@pytest.mark.parametrize(
+    "num_round",
+    [10, 100, 300],
+)
+@pytest.mark.parametrize(
+    "num_actors",
+    [1, num_cpus, None, modin.config.NPartitions.get() + 1],
+)
+@pytest.mark.parametrize(
+    "data",
+    [
+        (
+            load_breast_cancer(),
+            {"objective": "binary:logistic", "eval_metric": ["logloss", "error"]},
+        ),
+    ],
+    ids=[
+        "load_breast_cancer",
+    ],
+)
```

Comment:
This parameter has only one value in list, so it should be moved to the body of test I think.

Reply:
I think that in future this decision will be convenient if we want expansion set of datasets.

Reply:
data is set inside test